### ISSUE TITLE
Blog post: How do use vale with RTD

### ISSUE THREAD

**Issue by ericholscher** [readthedocs/website#274]
We should eventually add a linting step to builds, but lots of people want "linting", we should help them figure this out.

**Comment by humitos** [comment]
We've got a PR to setup codespell in our repository in https://github.com/readthedocs/readthedocs.org/pull/11195. Maybe something to consider here as well.

**Comment by humitos** [comment]
One thing that I was thinking about this was: "do we want to run these lintings on Read the Docs and _fail_ the build because of that?". I remember we used to have Sphinx's `lincheck` in our build workflow and we got tired of it because the build failed and we weren't able to see the preview or similar.

Maybe the vale/codespell check is a little different since with `linkcheck` we don't have control on the external links and it's a bit tedious that your build fails because something you didn't touch. However, spell checks are more related with the actual PR itself where the user has 100% control over it.

Thoughts?

**Comment by ericholscher** [comment]
I think it would work similar to our other build steps, where a non-zero exit code would fail the build.

**Comment by humitos** [comment]
Right, that's my point. Do we want to _fail the whole build_ just because a typo? Would that be a good UX?

**Comment by ericholscher** [comment]
Presumably the tool would allow you to configure this?
